addons/base/views.py

```diff
@@ -311,14 +312,24 @@ def create_waterbutler_log(payload, **kwargs):
     with transaction.atomic():
         try:
             auth = payload['auth']
-            # Don't log download actions
+            # Don't log download actions, but do update analytics
+            user = OSFUser.load(auth['id'])
             if payload['action'] in ('download_file', 'download_zip'):
+                node = AbstractNode.load(payload['metadata']['nid'])
+                if not node.is_contributor(user):
+                    url = furl.furl(payload['request_metadata']['request']['url'])
```

Comment:
The WB-side PR needed for this has been merged, but one change will be needed.  The request url is now in the `payload['request_meta']['url']` key (along with `referrer`, `user_agent`, and `method`).